common/Cargo.toml

```
@@ -3,4 +3,5 @@
 [workspace]
 members = [
 	"sealing",
+	"user-auth"
```

```suggestion
	"user-auth",
```
make diffs more pleasant to read :)